packages/@ember/routing/route.ts

```diff
@@ -1251,8 +1251,9 @@ class Route<Model = unknown> extends EmberObject.extend(ActionHandler, Evented)
       {
         id: 'deprecate-implicit-route-model',
         for: 'ember-source',
-        since: { available: '5.3.0' },
+        since: { available: '5.3.0', enabled: '5.3.0' },
```

Comment:
Added `enabled` as well, since it will be enabled right away when released I think.